src/CMakeLists.txt

```diff
@@ -484,7 +484,7 @@ foreach(type ${RUST_LIBS})
       # see src/rust/bitbox02-rust-c/src/lib.rs.
       # https://github.com/rust-lang/rust/issues/66740
       RUSTC_BOOTSTRAP=1
-      ${CARGO} build -v --features target-${type} --target-dir ${RUST_BINARY_DIR} ${RUST_CARGO_FLAG} ${RUST_TARGET_ARCH_ARG}
```

Comment:
I actually like seeing exact commands. Do you find it too verbose?
Also, doesn't seem to be related to prost protobufs or does it?

Reply:
Was accidentally added in https://github.com/digitalbitbox/bitbox02-firmware/pull/522 - can put it back, I don't mind either way.